Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
@ilovezfs I was able to reproduce your issue completly outside of homebrew. The grafana issue is caused by an incompatibility of yarn and npm@5. I would say running npm install on a dep tree created by yarn was never supported, it just accidently worked for npm@4 and now it's broken in npm@5. This will mostly likely need a fix in the grafana formula itself.